src/script/cpp_api/s_env.cpp

```diff
@@ -150,13 +150,19 @@ void ScriptApiEnv::initializeEnvironment(ServerEnvironment *env)
 
 		bool simple_catch_up = true;
 		getboolfield(L, current_abm, "catch_up", simple_catch_up);
+		
+		int min_y = -31000;
```

Comment:
I know that the MT world ends earlier but this should simply be `INT16_MIN` to not make any assumptions